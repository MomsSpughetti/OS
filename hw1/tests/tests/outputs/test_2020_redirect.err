cat: new: No such file or directory
cat: new: No such file or directory
cat: new: No such file or directory
smash error: kill: invalid arguments
smash error: kill: invalid arguments
smash error: open failed: No such file or directory
ls: cannot access '>/tmp/smash/test/new': No such file or directory
echo: write error: Bad file descriptor
cat: standard output: Bad file descriptor
smash error: chmod failed: No such file or directory
smash error: close failed: Bad file descriptor
rm: cannot remove 'new': No such file or directory
ls: write error: Bad file descriptor
smash error: execvp failed: No such file or directory
cat: standard output: Bad file descriptor
ls: write error: Bad file descriptor
rm: cannot remove 'new': No such file or directory
smash error: execvp failed: No such file or directory
cat: standard output: Bad file descriptor
ls: write error: Bad file descriptor
rm: cannot remove 'new': No such file or directory
ls: cannot access '>new': No such file or directory
cat: standard output: Bad file descriptor
smash error: close failed: Bad file descriptor
cat: standard output: Bad file descriptor
smash error: close failed: Bad file descriptor
cat: standard output: Bad file descriptor
