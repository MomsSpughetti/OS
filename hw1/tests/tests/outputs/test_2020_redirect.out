smash> smash> smash> smash> test_redirect> new
smash> smash> again_redirect> new
smash> smash> test_redirect>> new
smash> smash> smash> smash pid is 9249
smash> smash> smash pid is 9249
smash pid is 9249
smash> smash> smash pid is 9249
smash> mypid
smash> smash> smash> [1] sleep 100& : 9290 0 secs
smash> smash> smash> smash> smash> k
mypid
s
t
smash> smash> k
mypid
p
s
t
smash> k
mypid
p
s
t
smash> 